test/integration/containerized/test_container_management.py

```diff
@@ -70,3 +73,73 @@ def test_cancel_will_remove_container(test_data_dir, container_runtime_installed
     assert not is_running(
         cli, container_runtime_installed, 'ansible_runner_foo_bar'
     ), 'Found a running container, they should have all been stopped'
+
+
+@pytest.mark.skipif(shutil.which('podman') is None and shutil.which('docker') is None, reason='podman and docker are not available')
+@pytest.mark.parametrize('runtime', ['podman', 'docker'])
+def test_invalid_registry_host(tmp_path, runtime):
```

Comment:
This will be skipped unless both `podman` and `docker` are available, which is not the case in our current CI. Ideally we would test 

Since the change now works with `podman` and `docker`, you could use the `container_runtime_installed` fixture to get one of the available runtimes. There would still be a gap in coverage, but it's better than skipping the test completely.
```suggestion
def test_invalid_registry_host(tmp_path, runtime, container_runtime_installed):
```

Or you could skip inside the test.
```suggestion
@pytest.mark.parametrize('runtime', ['podman', 'docker'])
def test_invalid_registry_host(tmp_path, runtime):
    if shutil.which(runtime) is None:
        pytest.skip(f'{runtime} is unavaialble')
```

That might be better since it shows the skipped tests.